arcade/gui/constructs.py

```diff
@@ -10,13 +13,16 @@
 
 class UIMessageBox(UIMouseFilterMixin, UIAnchorLayout):
     """
-    A simple dialog box that pops up a message with buttons to close.
+    A simple dialog box that displays a message to the user.
 
-    :param width: Width of the message box
-    :param height: Height of the message box
-    :param message_text:
-    :param buttons: List of strings, which are shown as buttons
-    :param callback: Callback function, will receive the text of the clicked button
+    :param width float: The width of the message box.
+    :param height float: The height of the message box.
+    :param message_text str: The message text to display.
+    :param buttons tuple[str, ...]: List of strings, which are shown as buttons.
+    :param disappear bool: Whether the box should disappear after a set time or not.
+    :param disappear_time float: The time before the box should disappear.
+    :param callback Callable: The callback function which will receive the text of the
```

Comment:
Better than the previous drafts, but the syntax for type annotated params is the other way around: `:param float width:`